### PR TITLE
Adding temporary mitigation steps for WSO2-2021-1738.

### PR DESCRIPTION
**Pull request by anjai94**

### CODE REVIEW

**Review by ayomawdb — DISMISSED** [review]

File: en/docs/security-announcements/security-advisories/2023/WSO2-2022-2055.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+* WSO2 IS as Key Manager : 5.10.0
+* WSO2 Identity Server : 5.10.0
+
+
+### OVERVIEW
+XML external entity (XXE) vulnerability through SOAP API.
+
+### DESCRIPTION
+XML external entity (XXE) vulnerability is possible through SOAP API when the malicious actor is provided with
+"/permission/admin/manage/event-publish" permission.
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2023/WSO2-2022-2055.md:29]:
```suggestion
```

File: en/docs/security-announcements/security-advisories/2023/WSO2-2022-2055.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+* WSO2 API Manager : 3.1.0
+* WSO2 IS as Key Manager : 5.10.0
+* WSO2 Identity Server : 5.10.0
+
+
+### OVERVIEW
+XML external entity (XXE) vulnerability through SOAP API.
+
+### DESCRIPTION
+XML external entity (XXE) vulnerability is possible through SOAP API when the malicious actor is provided with
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2023/WSO2-2022-2055.md:28]:
```suggestion
XML external entity (XXE) vulnerability is possible through SOAP API when the malicious actor is provided with **/permission/admin/manage/event-publish** permission.
```

File: en/docs/security-announcements/security-advisories/2023/WSO2-2022-2177.md [hunk 3]
```diff
@@ -101,7 +101,7 @@ You may migrate to the latest version of the product if the latest version is no
 
 Or else you may follow the mitigation steps given below.
 
-#### Group 1:
+#### GROUP 1:
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2023/WSO2-2022-2177.md:104]:
```suggestion
#### GROUP 1
```

**Review by 1uffyD9 — CHANGES_REQUESTED** [review]
Suggested few changes.

**Review by 1uffyD9 — CHANGES_REQUESTED** [review]
Suggested few changes.

**Review by ayomawdb — APPROVED** [review]

**Pull request merged**